Formula/rex.rb

```diff
@@ -217,22 +232,23 @@ def install
 
     perl_build
     (libexec/"lib").install "blib/lib/Rex", "blib/lib/Rex.pm"
-
-    # Use system Perl
-    inreplace("bin/rexify") { |s| s.gsub!(/^#!perl$/, "#!/usr/bin/env perl") }
+    inreplace "bin/rex", "#!/usr/bin/env perl", "#!/usr/bin/perl"
```

Comment:
@DomT4 What was the reason behind this? When I tried running `rex -v`, it threw up:

~~~
Usage: DynaLoader::dl_find_symbol(libhandle, symbolname) at /usr/local/Cellar/rex/1.5.0_1/libexec/lib/perl5/darwin-thread-multi-2level/XSLoader.pm line 89.
Compilation failed in require at /usr/local/Cellar/rex/1.5.0_1/libexec/lib/Rex/CLI.pm line 18.
BEGIN failed--compilation aborted at /usr/local/Cellar/rex/1.5.0_1/libexec/lib/Rex/CLI.pm line 18.
Compilation failed in require at /usr/local/Cellar/rex/1.5.0_1/libexec/rex line 21.
BEGIN failed--compilation aborted at /usr/local/Cellar/rex/1.5.0_1/libexec/rex line 21.
~~~

Which is what I usually see if I run a Perl program using the system's version...